_docs/_latest/develop/dev/setup.md

```diff
@@ -92,7 +92,9 @@ to create a fork of the Forseti code, and learn how to submit a pull request (PR
 Use the following command to install required build dependencies:
 
   ```bash
-  pip3 install -q --upgrade -r forseti-security/requirements.txt
+  cd forseti-security
+
+  pip3 install -q --upgrade -r requirements.txt
```

Comment:
What's the benefit of this command vs the old one? they seem to do the same thing?

Reply:
`requirements.txt` is not a directory. Hence path is incorrect.

Reply:
but the previous command specified the path to requirements.txt including the directory i.e. `forseti-security/requirements.txt`

Reply:
![image](https://user-images.githubusercontent.com/41486977/62496776-ff2be900-b7a7-11e9-9ce8-de7e5be046f0.png)

Reply:
`requirements.txt` is not a directory. That's why path is incorrect.

Reply:
but if you were to run `pip3 install -q --upgrade -r forseti-security/requirements.txt` command, it should work, is there any reason why we need that change?

Reply:
Please share the screenshot if that command is working for you.

Reply:
![image](https://user-images.githubusercontent.com/5330393/62497230-3098ba80-b790-11e9-89cd-3f11017e106e.png)

Reply:
It needs to be updated for two reasons:
1. Command is working, but the path to `requirements.txt` is incorrect.
2. It is inconsistent with the `python3 setup.py install` command below `pip3 install -q --upgrade -r forseti-security/requirements.txt` command as shown in screenshot below.

![image](https://user-images.githubusercontent.com/41486977/62497549-5337cd00-b7aa-11e9-8e2f-ea964b5b992b.png)

Reply:
The command below `Running the Python setup` asked you to `cd` to the `forseti-security` folder, if you follow the instruction fully I don't see why it would fail though. 

@hshin-g @blueandgold any thoughts on this?

Reply:
Checkout most recent commit